query/src/sessions/session_mgr.rs

```diff
@@ -51,12 +57,14 @@ pub struct SessionManager {
     pub(in crate::sessions) max_sessions: usize,
     pub(in crate::sessions) active_sessions: Arc<RwLock<HashMap<String, Arc<Session>>>>,
     pub(in crate::sessions) storage_cache_manager: Arc<CacheManager>,
+    storage_accessor: Arc<dyn Accessor>,
```

Comment:
It's better to host an `Operator` here.

Reply:
We can do that here, reasons:
operator needs storage ctx, and the storage ctx is binding to query context for get metrics for each query.

Reply:
The ABI is `pub async fn get_storage_accessor(&self) -> **Result<Operator>**`, and `Operator::new(accessor)` almost no cost here, so I put them here.

Reply:
Update:
Change accessor to operator.